Sources/mParticle-Appboy/MPKitAppboy.m

```diff
@@ -15,6 +15,7 @@
 static NSString *const userIdTypeKey = @"userIdentificationType";
 static NSString *const emailIdTypeKey = @"emailIdentificationType";
 static NSString *const enableTypeDetectionKey = @"enableTypeDetection";
+static NSString *const forwardEnhancedCommerceDataKey = @"forwardEnhancedECommerceData";
```

Comment:
There was discussion to change the config setting.  Just pointing out that once that has been finalized, it will need to change here as well.